lib/build.js

```diff
@@ -8,7 +8,16 @@ const pify = require('pify');
 const writeFile = pify(fs.writeFile);
 
 module.exports = context => {
-	const dockerfile = `FROM node:${context.version}-onbuild`;
+	const dockerfile = [
+		`FROM node:${context.version}`,
+		'RUN mkdir -p /usr/src/app',
+		'WORKDIR /usr/src/app',
+		'COPY package.json /usr/src/app/',
+		'RUN npm install && npm cache clean --force',
+		'COPY . /usr/src/app',
+		'CMD [ "npm", "start" ]'
```

Comment:
Whitespace between `[` and `]` is unnecessary.

Reply:
Removed the whitespace. Although after looking at it again I wonder if it should be changed to `CMD ["npm, "test"]`. I initially implemented it as `CMD ["npm", "start"]` because I was emulating the behavior in the `node-onbuild` [Dockerfile](https://github.com/nodejs/docker-node/blob/master/8/onbuild/Dockerfile#L12) but I need to look into how the Docker `onbuild` images work again because I've been away from this code too long and have forgoten.